archetypes/database-mp/src/main/resources/README.md.mustache

```diff
@@ -38,4 +38,54 @@ curl -s -X GET http://localhost:8080/metrics
 curl -H 'Accept: application/json' -X GET http://localhost:8080/metrics
 {"base":...
 . . .
-```
\ No newline at end of file
+```
+
+## GraalVM Native Support
+
+The generation of native binaries requires an installation of GraalVM 20.0.0+. For more
+information about the steps necessary to use GraalVM with Helidon
+see https://helidon.io/docs/v2/#/mp/guides/36_graalnative.
+
+The H2 Database when configured to use the in-memory mode is currently _not compatible_
+with GraalVM native.
+In order to produce a native binary, you must run the H2 Database as a separate process
+and use a network connection for access. The simplest way to do this is by starting a Docker
+container as follows:
+
+```
+docker run -d -p 1521:1521 -p 81:81 -e H2_OPTIONS='-ifNotExists' --name=h2 oscarfonts/h2
+```
+
+The resulting container will listen to port 1521 for network connections.
+Switch property `javax.sql.DataSource.test.dataSource.url` in `microprofile-config.properties`
+to use a TCP connection:
+
+```
+javax.sql.DataSource.test.dataSource.url=jdbc:h2:tcp://localhost:1521/test;IFEXISTS=FALSE
+```
+
+Next, uncomment the following dependencies in your project's pom file:
+
+```
+<dependency>
```

Comment:
The first dependency is for Oracle JDBC, not needed for h2

Reply:
Ok, got it.